ci/azure-pipelines.yml

```diff
@@ -441,6 +441,9 @@ stages:
             Python310:
               python.version: '3.10'
               min_constraints: 'ci/constraints-test/py310-min-install.txt'
+            AirflowMin:
+              python.version: '3.7'
+              min_constraints: 'ci/constraints-test/py37-min-airflow.txt'
```

Comment:
@billdirks 
This is the more comprehensive set of tests that should run in the async pipeline

Reply:
I wonder if we can combine this strategy with the other `3.7` strategy so we don't have to run all these tests twice.

Reply:
@billdirks 
Done